### PR TITLE
Terraform Debugging instructions

### PR DESCRIPTION
**Pull request by kasparAxual**
Terraform Debugging instructions

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -92,4 +92,14 @@ go generate
...
 
 - **version** is the numeric version of the manifest format, not the version of our provider.
-- **protocol_versions** is the Terraform protocol version. This is set to 6.0 because we are using Terraform Plugin Framework to develop our provider.
\ No newline at end of file
+- **protocol_versions** is the Terraform protocol version. This is set to 6.0 because we are using Terraform Plugin Framework to develop our provider.
+
+## Debug
+- There are 4 IntelliJ IDEA run configurations in .run
+- Run "Axual build to _bin" to generate executable in /go/bin and name it terraform-provider-axual
+- Run "Axual Delve binary"(Run icon) to start Delve's debugging session on this binary. It looks for terraform-provider-axual executable in /go/bin.
```
Comment (yunussezgin) [inline readme.md:100]: We need a step to install Delve.
`brew install delve`

Comment (kasparAxual) [reply readme.md]: Done

**Review by yunusaxual — APPROVED** [review]
It is good to me. I followed the steps. And I was able to debug terraform.

File: readme.md [hunk 2]
```diff
@@ -92,4 +92,16 @@ go generate
 ## Terraform Manifest
 
 - **version** is the numeric version of the manifest format, not the version of our provider.
-- **protocol_versions** is the Terraform protocol version. This is set to 6.0 because we are using Terraform Plugin Framework to develop our provider.
\ No newline at end of file
+- **protocol_versions** is the Terraform protocol version. This is set to 6.0 because we are using Terraform Plugin Framework to develop our provider.
+
+## Debug
```
[the remark this answers is not in the document]
Comment (kasparAxual) [reply readme.md]: Added that these are IntelliJ IDEA specific instructions.
[Delve is a debugger for the Go programming language.](https://github.com/go-delve/delve)
It is not MacOS specific

**Review by markaxual — APPROVED** [review]

**Pull request merged**